src/scss/components/optional/_ratings.scss

```diff
@@ -10,11 +10,12 @@
  * https://fozzie.just-eat.com/styleguide/ui-components/custom/star-ratings
```

Comment:
Can you make sure documentation is added to the component library for these modifiers too?